content/en/docs/reference/config/policy-and-telemetry/mixer-overview/index.md

```diff
@@ -43,7 +43,7 @@ At a high level, Mixer provides:
 
 Policy enforcement and telemetry collection are entirely driven from configuration.
```

Comment:
@frankbu is grabbing the reference documentation part of the release process? I can't find any of this content in the API repository.

Cheers
-steve